src/components/dashboard/SendLinkSummary.js

```diff
@@ -124,6 +124,13 @@ const SendLinkSummary = ({ screenProps }: AmountProps) => {
           }
           log.debug('generateLinkAndSend: enqueueTX', { transactionEvent })
           userStorage.enqueueTX(transactionEvent)
+          if (Config.isEToro) {
+            userStorage.saveSurveyDetails(hash, {
+              reason,
+              amount,
+              survey: 'other',
```

Comment:
this shouldn't be hard coded. use a variable where you expect the survey results to be at.
anyone implementing the survey screen should use that variable